criu/include/criu-plugin.h

```diff
@@ -62,6 +62,8 @@ enum {
 
 	CR_PLUGIN_HOOK__POST_FORKING = 12,
 
+	CR_PLUGIN_HOOK__RESTORE_INIT = 13,
```

Comment:
It would be good to update [plugins/amdgpu/README.md](https://github.com/checkpoint-restore/criu/blob/a000d89c70611e83dcc113ae59d0e4fdd244237d/plugins/amdgpu/README.md) with information about the new plugin callbacks.